README.md

```diff
@@ -120,30 +130,29 @@ storyblokInit({
 
 `gatsby-source-storyblok` does three actions when you initialize it:
 
-- Provides a `useStoryblokState` custom hook in your app, that parses story content JSON into the object.
+- Provides a `StoryblokStory` that includes `useStoryblokState` that receives a story object.
 - Loads [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=gatsby-source-storyblok) for real-time visual updates.
 - Provides a `storyblokEditable` function to link editable components to the Storyblok Visual Editor.
 
-#### 1. Fetching Content
+> [gatsby-source-storyblok v5](https://github.com/storyblok/gatsby-source-storyblok/tree/v5) and lower had `useStoryblokState` that parsed story content JSON into the object. Use `StoryblokStory` instead to handle the Visual Editor live events when editing story.
```

Comment:
Shall this text be about only lower than v5 versions? Or v5 is also using the old `useStoryblokState`?

Reply:
@Dawntraoz Gatsby SDK v5 uses `useStoryblokState` 👍